frontend/src/components/SongList/SongList.tsx

```diff
@@ -0,0 +1,53 @@
+import { Song } from "../../models/song"
+import { ISpotifyUser } from "../App/App";
+import React, { useEffect, useState } from "react";
+// import { ProgressBar } from '@bit/primefaces.primereact.progressbar'
+import { Button } from "primereact/button";
+import { Accordion } from "../Accordion/Accordion"
+import "./SongList.css"
+
+interface ISongListProps {
+    songs: Song[];
+    song: Song;
+    setSong: (song: Song) => void;
+}
+
+interface ISongList {
+    spotifyUser?: ISpotifyUser;
+}
+interface ISongListState {
+    songs: Song[];
+    showList: boolean;
+}
+
+export const SongList: React.FC<ISongList & ISongListProps> = (props: ISongListProps) => {
+    const [songs, setSongs] = useState(props.songs);
+    // const [song, setSong] = useState(props.song)
+    const [showList, setShowList] = useState(false);//???????
```

Comment:
?????

Reply:
i completely forget why I added the question marks